README.md

```diff
@@ -343,16 +343,18 @@ extra_docker_arguments: []  # Additional arguments for `docker container create`
 
 The [toastfile](https://github.com/stepchowfun/toast/blob/main/toast.yml) for Toast itself is a comprehensive real-world example.
 
-## Cache configuration
+## Configuration
 
-Toast supports local and remote caching. By default, only local caching is enabled. Remote caching requires that the Docker Engine is logged into a Docker registry (e.g., via `docker login`).
-
-The caching behavior can be customized with a configuration file. The default location of the configuration file depends on the operating system:
+The behavior can be customized with a configuration file. The default location of the configuration file depends on the operating system:
```

Comment:
I think it would read more naturally if it just said "Toast can be customized..." (instead of "The behavior...").